.gitignore

```diff
@@ -1,2 +1,10 @@
 command_history
 local.config
+
+# exclude any new worktrees added
+MPAS-Model/*
```

Comment:
This is really nice. It didn't sink in that we could have multiple worktrees at this level within a compass repo. Thanks for thinking of this as a workflow.